Podfile

```diff
@@ -47,7 +47,7 @@ target 'WordPress', :exclusive => true do
   pod 'WPMediaPicker', '~> 0.9.0'
   pod 'WordPress-iOS-Editor', '1.2'
   pod 'WordPress-iOS-Shared', '0.5.3'
-  pod 'WordPressApi', :git => "https://github.com/wordpress-mobile/WordPress-API-iOS.git"
+  pod 'WordPressApi', '~>0.4'
```

Comment:
Actually I take that back - we're the most restrictive in WPiOS so I'd lock it solid against 0.4.0.